doc/integrator/authentication.rst

```diff
@@ -27,6 +27,19 @@ table. If a c2cgeoportal application should work with another user information
 source, like LDAP, a custom *client validation* mechanism can be set up.
 Our knowledge base has an example of how this can be achieved.
 
+Basic auth
+~~~~~~~~~~
+
+To be able acces to the OGC services from your desktop GIS you should anable the basic authentication
```

Comment:
To be able to access the OGC services from your desktop GIS, you should enable the basic authentication